src/server/saved_objects/client/saved_objects_client.js

```diff
@@ -175,26 +178,62 @@ export class SavedObjectsClient {
       return { saved_objects: [] };
     }
 
-    const docs = objects.map(doc => {
-      return { _type: get(doc, 'type'), _id: get(doc, 'id') };
-    });
+    const docs = objects.reduce((acc, { type, id }) => {
+      acc.push({}, createIdQuery({ type, id }));
+      return acc;
```

Comment:
Rather than mutating `acc`, how about using spread:

```js
return [...acc, {}, createIdQuery({ type, id })];
```

That suggestion out of the way, why push an empty object here (ie the first arg)?